happytransformer/happy_word_prediction.py

```diff
@@ -60,6 +65,10 @@ def predict_mask(self, text, targets=None, top_k=1):
             for answer in result:
                 if answer["token_str"][0] == "▁":
                     answer["token_str"] = answer["token_str"][1:]
+        elif self.model_type == "ROBERTA":
+            for answer in result:
+                if answer["token_str"][0] == "Ġ":
+                    answer["token_str"] = answer["token_str"][1:]
```

Comment:
I'm having scalability concerns. We need to decide on where we are going to put this kind of model-specific logic before our base classes get too contrived. 

My intuition wants an `Adapter` for each class that our user-facing classes grab under the hood, that would include the (few) implementation-specific concerns we have

Looks good tho, you followed the pattern we were already using for ALBERT.

I'll probably address this in a separate PR, but I'm interested in thoughts anyone has here.

Reply:
I agree we should look into making something a little more scalable. For now a couple if/elif statements is fine.

Reply:
When can I do that without treading on toes? Are there any other models we're aiming to add for v2 release?

Reply:
Is this something that you'd like me to address right now, or is @ted537 taking on this task (in relation to comment above)?

Reply:
I missed this comment which is why I'm just replying now. 

I think we're good for models for the first release of v2